bazel/grpc_deps.bzl

```diff
@@ -519,3 +520,11 @@ def grpc_test_only_deps():
             ],
             build_file = "@com_github_grpc_grpc//third_party:constantly.BUILD",
         )
+
+    if "com_google_libprotobuf-mutator" not in native.existing_rules():
```

Comment:
nit: for consistency, you can use `com_google_libprotobuf_mutator` as a name (change dash into underscore). That's consistent with naming of other dependencies in this file that also have a dash in their name.

Reply:
done